src/core/init/load_on_media_source.ts

```diff
@@ -210,7 +208,8 @@ export default function createMediaSourceLoader(
         return observableOf(EVENTS.loaded(segmentBuffersStore));
       }));
 
-    return observableMerge(loadedEvent$,
+    return observableMerge(durationUpdater$,
```

Comment:
It has to be noted that here, as I understand it, the first duration assignment will be done at subscription. It was not the case before, are we sure that it is not risky ?